### DISCUSSION TITLE
Correct application of changes patroni.yml

### DISCUSSION THREAD

**Discussion by shipiguzev** [autobase-tech/autobase#123] [Q&A]
Hi,

The /etc/patroni/patroni.yml contain the current configuration postgresql after executing deploy_pgcluster.yml --tag patroni_conf,
but i haven't seen changes in patronictl show-config. After restart service patroni i haven't seen changes too.
How should I apply the changes correctly?

**Comment by vitabaks** [comment]
Hi @shipiguzev 

For parameters that are stored in DCS use `patronictl edit-config`

P.S.
I recommend also watch this video: [Tutorial: Management of High-Availability PostgreSQL clusters with Patroni](https://pgconf.ru/en/2018/108567)
